src/Content/Conversation.php

```diff
@@ -985,7 +985,8 @@ private function addChildren(array $parents, bool $block_authors, string $order,
 
 		$thread_items = Post::selectForUser($uid, array_merge(ItemModel::DISPLAY_FIELDLIST, ['featured', 'contact-uid', 'gravity', 'post-type', 'post-reason']), $condition, $params);
 
-		$items = [];
+		$items       = [];
+		$quoteuriids = [];
```

Comment:
Please use either `camelCase` or `snake_case` for variable names.
```suggestion
		$quoteUriIds = [];
```

Reply:
Done